app/lib/bk/compat/parsers/gha/expressions.rb

```diff
@@ -138,6 +138,10 @@ def self.replace_context_vars(var_name)
         "$#{var_name}"
       end
 
+      def self.replace_context_matrix(var_name)
+        var_name.gsub(/\$\{\{\s*(matrix\.[\w.-]+|matrix)\s*\}\}/, '{{\1}}')
```

Comment:
I think all it needs is actually just
```suggestion
        var_name
```

Reply:
Funnily enough, that's what my local change is/was before I headed off.